### DISCUSSION TITLE
Training loss Oscillating

### DISCUSSION THREAD

**Discussion by RadeenXALNW** [facebookresearch/detectron2#5021] [General]
Hey. For improving the learning capabilities I have added augmentation in detectron2 maskrcnn 50 FPN. 
My learning rate decrease but no that much. Most of the time it oscillates from 1.5 to 1.6. At first it drastically decrease (from 2.8 to 1.7 in just few epochs) than just oscillates and decrease very slightly. I tried to change the batch size 8 16 32.
